eo-maven-plugin/src/main/java/org/eolang/maven/objectionary/OyRemote.java

```diff
@@ -71,6 +73,22 @@ public Input get(final String name) throws MalformedURLException {
         return new InputOf(url);
     }
 
+    @Override
+    public boolean contains(final String name) throws MalformedURLException {
```

Comment:
@MikhailLipanin We need test for that method.

Reply:
done